configs/config-1.26/base.yaml

```diff
@@ -25,6 +25,7 @@ packages:
 - shim
 - efibootmgr
 - iscsi-initiator-utils
+- kernel-uek-modules-5.15.0
```

Comment:
Unless this doesn't work without this version, you should avoid using it.  It will change how the rpm solver works and can cause you to silently not pick up newer versions of the kernel.

When it comes to RPMs, the best bet is to have as few constraints as possible and only use them when it is necessary.